Rename ai2thor experiments to cups rl (#9)
TheMTank/ai2thor-experiments: README.md

> # cups-rl

Reviewer: Change title to `cups-rl -- Customizable Unity Photorrealistic Simulation for Reinforcement Learning algorithms`  or something like that. If not, mention it somewhere in the intro

Author: done

within an openAI gym interface, so that new and more complex tasks can be developed efficiently to 
train and test the agent.

> We will release a blog soon which will go into more detail about this repo and the first results of algorithms trained on these tasks.

Reviewer: ```suggestion
We will release a blog soon which will go into more detail about this repo and how to exploit all of its features on these tasks.
```

Reviewer: Or sth in those lines. Algorithms comes later

Author: and how to use all of the features of our wrapper.

## Installation

Clone cups-rl repository:

```

> # AI2THOR_EXPERIMENTS=/path/to/clone/cups-rl

Reviewer: ```suggestion
# CUPS_RL=/path/to/clone/cups-rl
```

git clone https://github.com/TheMTank/cups-rl.git $AI2THOR_EXPERIMENTS
```

> Install Python dependencies (Currently only supporting python 3.5+):

Reviewer: below this change the AI2THOR_EXPERIMENTS variable to CUPS_RL

  <p>Example of task "place cup in microwave"</p>
</div>

<div align="center">
  <img src="docs/bowls_fp_404_compressed_gif.gif" width="294px" />

>   <p>Semi-optimal A3C agent learning on NaturalLanguagePickUpMultipleObjectTask with the object being CUPS!</p>

Reviewer: ```suggestion
  <p>A3C agent learning during training on NaturalLanguagePickUpMultipleObjectTask in one of our customized scenes with the object being CUPS!</p>
```

Reviewer: Also delete the first gif. I would like to have something original from us rather than the one from ai2thor. Worst case scenario you can create it in interactive mode

Author: i get your point about removing the microwave gif but i also like 2 gifs
how about, if you think of a 2nd gif we can replace the microwave gif?
